default.nix

```diff
@@ -18,7 +18,7 @@ in naersk.buildPackage {
   shellHook =
   ''
     echo "Hello shell"
```

Comment:
Can you remove that line please while you are at it? Must have slipped my review last time but this line doesn't do anything.